vets-website.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	]
+}
```

Comment:
Did we mean to commit this file? Looks like it's vscode-specific 🙂

Reply:
Revert inc - this existed before I jumped in, good catch.